src/prometheus/alert.rs

```diff
@@ -0,0 +1,64 @@
+use std::collections::HashMap;
+
+use serde::{Deserialize, Serialize};
+
+#[derive(Serialize, Deserialize, Debug, PartialEq, Eq)]
+pub struct Alerts {
+    pub groups: Vec<AlertGroup>,
+}
+
+#[derive(Serialize, Deserialize, Debug, PartialEq, Eq)]
+pub struct AlertGroup {
+    pub name: String,
+    pub rules: Vec<AlertRules>,
+}
+
+#[derive(Serialize, Deserialize, Debug, PartialEq, Eq)]
+pub struct AlertRules {
+    pub alert: String,
```

Comment:
what's the alert field for?

Reply:
Following the Prometheus structure here from their docs for the final output format, `alert` is the name of a single alert that is part of a group of alerts.